healthtools/config.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+AWS = {
+    "aws_access_key_id": os.getenv("MORPH_AWS_ACCESS_KEY_ID"),
+    "aws_secret_access_key": os.getenv("MORPH_AWS_SECRET_KEY"),
+    "region_name": os.getenv("MORPH_AWS_REGION"),
```

Comment:
Let's just default this to `eu-west-1`. We don't intend to use any other region in the near future.